### ISSUE TITLE
Better explanatory text for /people page

### ISSUE THREAD

**Issue by jywarren** [publiclab/plots2#991]
This has been marked as a good candidate for becoming a [first-timers-only issue](http://www.firsttimersonly.com) like [these](https://github.com/publiclab/plots2/labels/first-timers-only), meaning that it's simple, self-contained, and with some extra formatting, could be a great entry point for a new contributor. If you're familiar enough with this code, please consider [reformatting or reposting it as a first-timers-only issue](https://publiclab.org/notes/warren/10-31-2016/create-a-welcoming-first-timers-only-issue-to-invite-new-software-contributors), and then ping @jywarren to get it labelled.

****

The template at https://publiclab.org/people currently shows just a brief title and a list of recently active people, and to admins, a notice about moderating users.

The template is here:

https://github.com/publiclab/plots2/blob/master/app/views/users/list.html.erb#L3-L6

Just after that section, we could add a friendly message telling who these people are, and perhaps inviting people to join the site. Here's the text:

> This page shows a list of all Public Lab contributors and is sorted by those who have posted most recently. Click the profile link to view their posting activity.

**Comment by steviepubliclab** [comment]
where did the idea for putting /people on the menu come from? I think we
should continue our menu redesign talk before we start adding things on it.
This page isn't my favorite way of showing people and activity on the site,
I think the upcoming geography and dashboard activity will do this better.
It's sort of feels like an uncomfortable ranking list of users.

**Comment by jywarren** [comment]
It's a long standing request which I'm trying to get checked off the list. If that's not a priority, we can cancel the request and remove that addition from #990 (this issue is just for the explanatory text, not the menu). But please check with other staff before commenting on #990 to that effect, thanks!

**Comment by Pk13055** [comment]
I would like to work on this issue.

**Comment by jywarren** [comment]
Hi, @Pk13055 - we need some input from Stevie and Liz, linked to above, before we can move forward on this, so if you could take a look at other available issues at https://github.com/publiclab/plots2/labels/help-wanted or https://github.com/publiclab/plots2/labels/fto-candidates, that would be great, thank you!

**Comment by jywarren** [comment]
@steviepubliclab and @ebarry - need some text suggestions (see "add a friendly message" above) to make a clear next step, thank you!

**Comment by steviepubliclab** [comment]
Hi sure:

@ebarry how does this sound:

"This page shows a list of all Public Lab users and is sorted by those who
have posted most recently. Click the profile link to view the user's
posting activity.”

**Comment by jywarren** [comment]
@Pk13055- hi, this is ready to go now, we'd love to have your contribution if you're still interested, thanks!

**Comment by jywarren** [comment]
A pull request was submitted in #1041 and is under review, thanks!
